generators/Exercises/Acronym.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Generators.Exercises
+{
+    public class Acronym : EqualityExercise
+    {
+        public Acronym() : base("acronym")
```

Comment:
I like it! The PR has been updated. Don't merge it yet, as there is still a problem with the multi-line formatting in `beer-song`. I think that's the next thing I want to get right: figuring out how to best configure how the output is formatted.